examples/src/examples/graphics/lit-material.tsx

```diff
@@ -1,18 +1,13 @@
 import * as pc from '../../../../';
-class CustomMaterialExample {
+
+class LitMaterialExample {
     static CATEGORY = 'Graphics';
-    static NAME = 'Custom Material';
+    static NAME = 'Lit Material';
     static HIDDEN = true;
-    static WEBGPU_ENABLED = true;
+    static WEBGPU_ENABLED = false;
```

Comment:
is something not working on WebGPU?

Reply:
Yes the reflections are incorrect, not sure why:

<img width="1355" alt="lit-material-example" src="https://github.com/playcanvas/engine/assets/11276292/86dfb3e1-5a82-4dd3-b854-e4bac24df3ae">

Reply:
Perhaps wgpu is generating mipmaps when it shouldn't?

Reply:
Setting `mipmaps: false` solves this issue so I've re-enabled wgpu on the example.

It does bring up an interesting issue with the difference between webgl and webgpu initialisation. There will likely be more bugs like this as we transition between GL->GPU.

Reply:
where did you need to change the mipmaps?

Reply:
On the asset load options object. All the `helipad-env-atlas` assets were flagged with `mipmaps: false` except for two examples (including this one). I've updated both to specify the flag now.

BTW what I'm referring to with bugs is that the scene sets texture.mipmap=false when used as env-atlas. This works fine with WebGL because the texture is created later on, but under WebGPU this doesn't work.

Reply:
Yep ok, all good, glad it's just this, I log out a debug warning when these settings that are not supported to be changed later (currently anyways) are changed.